src/interface/RHInterface.py

```diff
@@ -6,8 +6,10 @@
 import gevent # For threads and timing
 from monotonic import monotonic # to capture read timing
 
+from Plugins import Plugins, search_modules
 from Node import Node
-from BaseHardwareInterface import BaseHardwareInterface, PeakNadirHistory, discover_modules, discover_plugins
+from BaseHardwareInterface import BaseHardwareInterface, PeakNadirHistory
+from xml.etree.ElementInclude import include
```

Comment:
Is `from xml.etree.ElementInclude import include` needed?

Reply:
Nope, pydev being unhelpful.